src/modules/universal-discovery/components/content-table/content.table.component.js

```diff
@@ -205,7 +205,7 @@ ContentTableComponent.propTypes = {
     perPage: PropTypes.number.isRequired,
     onItemSelect: PropTypes.func.isRequired,
     contentTypesMap: PropTypes.object.isRequired,
-    onItemClick: PropTypes.func.isRequired,
+    onItemClick: PropTypes.func,
```

Comment:
Not required anymore?

Reply:
By default it is `null`, thus it cannot be set as `isRequired`.